frontend_lib/src/component/Button/IconButton.jsx

```diff
@@ -37,13 +37,27 @@ const IconButton = props => {
           }}
         />
       )}
-      {props.text && <span className={props.icon ? 'iconbutton__text_with_icon' : 'iconbutton__text'}>{props.text}</span>}
+
+      {props.text && (
+        <span
+          className={`${props.icon ? 'iconbutton__text_with_icon':''} iconbutton__label`}>
```

Comment:
```suggestion
          className={`${props.icon ? 'iconbutton__text_with_icon': ''} iconbutton__label`}>
```

or you can use classnames too (don't forget to add the import), like
```suggestion
          className={classnames('iconbutton__label', { iconbutton__text_with_icon: props.icon })}>
```

Same suggestion for the other classnames.